components/event/EventCard.js

```diff
@@ -112,15 +108,15 @@ export default function EventCard({ manage, event, usernames }) {
                 </span>
               </p>
             </div>
-            <div className="isolate flex -space-x-1 ">
+            <div className="flex -space-x-1 isolate ">
               {usernames &&
                 usernames.map((username) => {
                   return (
                     <Link
                       href={`/${username}`}
                       key={username}
                       aria-label={`Visit user ${username}`}
-                      className="hidden lg:block h-10 w-10"
```

Comment:
this change looks the same?

Reply:
This change is done by my prettier. So, it is just making tailwind css properties arrange alphabetically.